doc/Mammal Gestation Summary.Rmd

```diff
@@ -0,0 +1,46 @@
+---
+title: "Mammal Gestation Summary"
+output: html_notebook
+---
+
+##Paper 1:
```

Comment:
Don't forget the space between `#` and the text. You don't need colons in headers:

    ## Paper 1